test/verify/machineslib.py

```diff
@@ -1436,6 +1436,61 @@ def installWithErrorTest(dialog):
                 .deleteVm(dialog) \
                 .checkEnvIsEmpty()
 
+        def setupMockFileServer():
+            self.machine.upload(["verify/files/min-openssl-config.cnf", "verify/files/mock-range-server.py"], "/tmp/")
```

Comment:
It's a bit confusing that this commit uses /tmp, and the next one immediately changed it. So the net result is that this PR uses vm_tmpdir.